### DISCUSSION TITLE
Script permissions

### DISCUSSION THREAD

**Discussion by JoelRobG** [derekantrican/GAS-ICS-Sync#329] [Q&A] [answered]
Just wondering why the script requires "Send email as you" permissions?

Thanks,
Joel

**Comment by jonas0b1011001** [comment, marked as answer]
There are options to 
1. [Notify you about new releases](https://github.com/derekantrican/GAS-ICS-Sync/blob/514c294a9daa7f761a6333192cde4e75c14f0630/Helpers.gs#L967)
2. [Send Summaries with added/modifies/deleted events](https://github.com/derekantrican/GAS-ICS-Sync/blob/514c294a9daa7f761a6333192cde4e75c14f0630/Helpers.gs#L875)

Both will send an email from your account to the address specified in [code.gs line 50](https://github.com/derekantrican/GAS-ICS-Sync/blob/514c294a9daa7f761a6333192cde4e75c14f0630/Code.gs#L50)

**Reply by derekantrican** [reply]
Adding to jonas' answer - these are optional features controlled behind the `emailSummary` variable. If you are paranoid, you can remove any lines that say `MailApp.sendEmail....` and it should not request this permission when authorizing.
